graphdatascience/tests/conftest.py

```diff
@@ -3,6 +3,9 @@
 
 def pytest_addoption(parser: Any) -> None:
     parser.addoption("--include-enterprise", action="store_true", help="include tests requring GDS enterprise")
+    parser.addoption(
```

Comment:
nit about the flag name: at least in Aura, all *DS databases uses encryption so, it's not really an "enterprise" feature.

Reply:
makes sense, we can call it `--encrypted`

Reply:
this is just a test flag right?